game/client/menus/main_menu.lua

```diff
@@ -29,9 +32,19 @@ local function onCreate(overlay)
     exitGame.onClick = function()
         game.control.shutdown()
     end
+
+    testCheck.onClick = function()
+        if(testCheck.checked) then
+            testCheck.checked = false
+            testCheck.image = checkboxTexture_unchecked
+        else
+            testCheck.checked = true
+            testCheck.image = checkboxTexture_checked
+        end
+    end
```

Comment:
This is a bit funky 😄 

The behavior of the checkbox should be there by default, defined in the CheckBoxWidget class.

Reply:
I made a mistake when I was looking it over, just missed your last comment. Thanks for the feedback! I'll get to work changing this